src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -25,170 +25,185 @@ const propTypes = {
-        }
+    if (values.ssnLast4 && !ValidationUtils.isValidSSNLastFour(values.ssnLast4)) {
+        errors.ssnLast4 = 'bankAccount.error.ssnLast4';
+    }
 
-        return errors;
+    if (values.requestorAddressStreet && !ValidationUtils.isValidAddress(values.requestorAddressStreet)) {
+        errors.requestorAddressStreet = 'bankAccount.error.addressStreet';
     }
 
-    submit(values) {
-        const payload = {
-            bankAccountID: lodashGet(this.props.reimbursementAccount, 'achData.bankAccountID') || 0,
-            ...values,
-        };
+    if (values.requestorAddressZipCode && !ValidationUtils.isValidZipCode(values.requestorAddressZipCode)) {
+        errors.requestorAddressZipCode = 'bankAccount.error.zipCode';
+    }
 
-        BankAccounts.updatePersonalInformationForBankAccount(payload);
+    if (!ValidationUtils.isRequiredFulfilled(values.isControllingOfficer)) {
+        errors.isControllingOfficer = 'requestorStep.isControllingOfficerError';
     }
 
-    render() {
-        if (this.props.shouldShowOnfido) {
-            return (
-                <RequestorOnfidoStep
-                    reimbursementAccount={this.props.reimbursementAccount}
-                    reimbursementAccountDraft={this.props.reimbursementAccountDraft}
-                    onBackButtonPress={this.props.onBackButtonPress}
-                    getDefaultStateForField={this.props.getDefaultStateForField}
-                />
-            );
-        }
+    return errors;
+};
 
+function InnerRequestorStep({reimbursementAccount, shouldShowOnfido, reimbursementAccountDraft, onBackButtonPress, getDefaultStateForField, translate}, ref) {
+    const defaultValues = useMemo(
+        () => ({
+            firstName: getDefaultStateForField('firstName'),
+            lastName: getDefaultStateForField('lastName'),
+            street: getDefaultStateForField('requestorAddressStreet'),
+            city: getDefaultStateForField('requestorAddressCity'),
+            state: getDefaultStateForField('requestorAddressState'),
+            zipCode: getDefaultStateForField('requestorAddressZipCode'),
+            dob: getDefaultStateForField('dob'),
+            ssnLast4: getDefaultStateForField('ssnLast4'),
+        }),
+        [getDefaultStateForField],
+    );
+
+    const submit = useCallback(
+        (values) => {
+            const payload = {
+                bankAccountID: lodashGet(reimbursementAccount, 'achData.bankAccountID') || 0,
+                ...values,
+            };
+
+            BankAccounts.updatePersonalInformationForBankAccount(payload);
+        },
+        [reimbursementAccount],
+    );
+
+    const LabelComponent = useCallback(
+        () => (
+            <View style={STYLES.Label}>
+                <Text>{translate('requestorStep.isControllingOfficer')}</Text>
+            </View>
+        ),
+        [translate],
+    );
+
+    if (shouldShowOnfido)
```

Comment:
Add the `{}` to the if, because it is in multiple lines, it hard to read

Reply:
Done ✅